app/models/player.rb

```diff
@@ -3,6 +3,7 @@ class Player < ApplicationRecord
 
   has_many :game_players, -> { order(position: :asc) }, dependent: :destroy, inverse_of: :player
   has_many :games, through: :game_players
+  has_many :rounds, dependent: :destroy
```

Comment:
Is the player directly associated with a round?

Reply:
deleted